packages/calcite-components/src/tests/commonTests/themed.ts

```diff
@@ -115,13 +121,14 @@ export function themed(componentTestSetup: ComponentTestSetup, tokens: Component
           styleTargets[selector][1].push(tokenStyle);
         }
         if (shadowSelector) {
-          target = shadowSelector ? await page.find(`${selector} >>> ${shadowSelector}`) : target;
+          const effectiveShadowSelector = shadowSelector.replace(/::.*$/, "");
+          target.el = await page.find(`${selector} >>> ${effectiveShadowSelector}`);
```

Comment:
I tested this with alert, action, and menu-item. Action passed but when I tested alert and menu-item it only passed when I put a debugger on line 125 and then immediately continued to play the test. 

```
const tokens: ComponentTestTokens = {
        "--calcite-alert-dismiss-progress-background-color": {
          shadowSelector: `.${CSS.dismissProgress}::after`,
          targetProp: "backgroundColor",
        },
      };
      themed(`<calcite-alert open auto-close>`, tokens);
```


```
const tokens: ComponentTestTokens = {
      "--calcite-menu-item-action-border-color": [
        {
          shadowSelector: "calcite-action::after",
          targetProp: "backgroundColor",
        },
      ],
    };
    themed(`calcite-menu-item`, tokens);
    ```

Reply:
This might be a separate issue since all updates in this PR are synchronous or update existing async code.

I noticed the PR was not synced up with `epic/7180-component-tokens`, which I have taken care of. Would you mind testing again?